src/features/game/expansion/lib/boosts.ts

```diff
@@ -177,7 +177,10 @@ export const getFoodExpBoost = (
     boostedExp = boostedExp.mul(1.2);
   }
 
-  if (isCollectibleBuilt({ name: getSeasonalBanner(), game })) {
+  if (
+    isCollectibleBuilt({ name: getSeasonalBanner(), game }) ||
```

Comment:
Recommend:
Add a new function `isBannerActive()` and use that where appropriate.